File: frontend/lib/model/activity_status.dart
import 'package:json_annotation/json_annotation.dart';

enum ActivityStatus {
  @JsonValue('READY')
  READY,
  @JsonValue('CLOSED')
  CLOSED,
  @JsonValue('DELETED')
  DELETED,
  @JsonValue('OVER')
  OVER,
}

File: src/main/java/com/venus/meetspace/model/enums/ActivityStatus.java
package com.venus.meetspace.model.enums;

import com.baomidou.mybatisplus.annotation.EnumValue;
import lombok.Getter;

@Getter
public enum ActivityStatus {

    READY(0),
    CLOSED(1),
    DELETED(2),
    OVER(3);

    @EnumValue
    private final int code;

    ActivityStatus(int code) {
        this.code = code;
    }
}
